duckduckgo/iOS
# Bump BSK to pull in C-S-S 6.19.0

Task/Issue URL: https://app.asana.com/0/72649045549333/1207836782368194/f
Tech Design URL: https://app.asana.com/0/72649045549333/1207836782368196/f
Corresponding BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/1006
Corresponding macOS PR: https://github.com/duckduckgo/macos-browser/pull/3347

CC: @jonathanKingston 

**Description**:
This PR bumps the BSK pin to pull in the changes from https://github.com/duckduckgo/content-scope-scripts/pull/1026.



**Steps to test this PR**:
1. Navigate to any old [MSN article page](https://www.msn.com/en-us/sports/nfl/how-saquon-barkley-gave-eagles-coach-a-hard-lesson-in-accountability-while-saving-season/ar-AA1qPl89?ocid=hpmsn&cvid=81d6bc9eb97e442ba7de62c248189b23&ei=12) and refresh the page two or more times. See that the Expand Article button persists across reloads.



**Definition of Done (Internal Only)**:


**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
